app/views/ReactionsView/styles.js

```diff
@@ -0,0 +1,43 @@
+import { StyleSheet } from 'react-native';
+
+import sharedStyles from '../Styles';
+
+export default StyleSheet.create({
+	...sharedStyles,
```

Comment:
We should not do this, because it's duplicating the whole style

Reply:
Removed